nucliadb/src/nucliadb/search/search/chat/prompt.py

```diff
@@ -524,10 +524,7 @@ async def _get_paragraph_text(
     ) -> tuple[ParagraphId, str]:
         return pid, await get_paragraph_text(
             kbid=kbid,
-            rid=pid.field_id.rid,
-            field=pid.field_id.full(),
```

Comment:
Here we pass the field as `full()` and https://github.com/nuclia/nucliadb/blob/349fa0c495a6c544b58c11b60ce1e5ef96be1a89/nucliadb/src/nucliadb/search/search/results_hydrator/base.py#L104 is using this id in the find response